ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/impl/FieldInfoImpl.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.api.client.impl;
+
+import io.confluent.ksql.api.client.ColumnType;
+import io.confluent.ksql.api.client.FieldInfo;
+import java.util.Objects;
+
+public final class FieldInfoImpl implements FieldInfo {
+
+  private final String name;
+  private final ColumnType type;
+  private final boolean isKey;
+
+  FieldInfoImpl(final String name, final ColumnType type, final boolean isKey) {
+    this.name = Objects.requireNonNull(name);
+    this.type = Objects.requireNonNull(type);
```

Comment:
Add the 2nd parameter here. i.e. `requireNonNul(name, "name")`. It helps troubleshoot the NPE errors. I had a problem in the past when I was getting NPE and only a line number appeared. Having the NPE print the String helps easily identify the error instead of a line number.

Reply:
Done.